components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -331,14 +331,16 @@ def render_thumbnail(request, iid, w=None, h=None, conn=None, _defcb=None,
     """
     server_id = request.session['connector'].server_id
     direct = True
+    defaultSize = conn.getDefaultThumbnailSize()
```

Comment:
This will slows down thumbnail loading, why do you need that value on each request? that should be set once when connection is created